docs/guides/caddy-configuration.md

```diff
@@ -0,0 +1,63 @@
+# Caddy Webserver Frontend
```

Comment:
Remove this header, as it will be automatically replaced with the page title in the mkdocs config.